Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -105,22 +105,24 @@ public override void OnPreSceneQuery()
 
         /// <summary>
         /// Gets the position of where grasp happens
-        /// For sixdof it's just the pointer origin
-        /// for hand it's the average of index and thumb.
+        /// For IMixedRealityHand it's the average of index and thumb.
+        /// For any other IMixedRealityController, return just the pointer origin
         /// </summary>
         public bool TryGetNearGraspPoint(out Vector3 result)
         {
-            // For now, assume that anything that is a sphere pointer is a hand
-            // TODO: have a way to determine if this is a fully articulated hand and return 
-            // ray origin if it's a sixdof
+            // If controller is of kind IMixedRealityHand, return average of index and thumb
             if (Controller != null && Controller is IMixedRealityHand)
             {
-                HandJointUtils.TryGetJointPose(TrackedHandJoint.IndexTip, Controller.ControllerHandedness, out MixedRealityPose index);
-                HandJointUtils.TryGetJointPose(TrackedHandJoint.ThumbTip, Controller.ControllerHandedness, out MixedRealityPose thumb);
-                if (index != null && thumb != null)
+                var hand = Controller as IMixedRealityHand;
```

Comment:
About my comment above about HandJointUtils.TryGetJoint(Controller). Seems like a great example of where that method would be useful.

Reply:
@julenka, these were instances of people calling HandJointUtils when they didn’t have to after doing a find all references